opensrp-anc/src/main/java/org/smartregister/anc/library/presenter/ProfilePresenter.java

```diff
@@ -24,16 +23,17 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import timber.log.Timber;
+
 /**
  * Created by ndegwamartin on 13/07/2018.
  */
 public class ProfilePresenter implements ProfileContract.Presenter, RegisterContract.InteractorCallBack {
-    private static final String TAG = ProfilePresenter.class.getCanonicalName();
-
     private WeakReference<ProfileContract.View> mProfileView;
     private ProfileContract.Interactor mProfileInteractor;
     private RegisterContract.Interactor mRegisterInteractor;
     private ContactInteractor contactInteractor;
+    private String baseEntityId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'baseEntityId' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-anc/pullRequest?prid=4657615)